handlers/author_editor.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def RemoveColon(string):
+    return string.replace(':', '')
+
+
+def author_editor_cleanup(df):
+    # remove last column, its empty
+    df.drop(df.columns[len(df.columns) - 1], axis=1, inplace=True)
+    # replace all \N with NaN (pandas like it this way)
+    df.replace("\\N", np.nan, inplace=True)
+    # some authors names have a : at the end, remove that
+    df['authors'] = df['authors'].astype(str).apply(RemoveColon)
+    df['editors'] = df['editors'].astype(str).apply(RemoveColon)
+    return df
```

Comment:
I think you missed the branch of @julianewundermann in which she worked on the clean up of the authors and editors.

see here: https://github.com/code-openness/Data/tree/7f933c45722947e7cbe97533a272c6379876f4cf

Reply:
OK, I was also wondering about that, the code is in "find_erroneous_data.ipynb", I will look into it. This may take a bit of time.

Reply:
I will merge this pull request, you can then create another one when you have the code ready